src/decode/mod.rs

```diff
@@ -367,7 +367,7 @@ mod tests {
 
         let mut v: Vec<u8> = vec![];
         let bytes_per_line = BASE64_PEM_WRAP * 3 / 4;
-        for _i in 0..2*bytes_per_line {
+        for _i in 0..2 * bytes_per_line {
```

Comment:
I actually think this is less clear since the precedence of `*` is before `..`.

Reply:
Agree, but it's what `cargo fmt` prefers. I'll put some brackets to clarify precedence.